Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -77,6 +77,50 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
         })
     }
 
+    override var keyCommands: [UIKeyCommand]? {
+        return [
+            UIKeyCommand(input: UIKeyInputLeftArrow, modifierFlags: .init(rawValue: 0), action: #selector(self.handleKeyCommand(sender:))),
+            UIKeyCommand(input: UIKeyInputRightArrow, modifierFlags: .init(rawValue: 0), action: #selector(self.handleKeyCommand(sender:)))
+        ]
+    }
+    
+    func handleKeyCommand(sender: UIKeyCommand) {
+        switch sender.input {
+        case UIKeyInputLeftArrow:
+            if completionActive {
+                applyCompletion()
+                
+                // Set the current position to the beginning of the text.
+                selectedTextRange = textRange(from: beginningOfDocument, to: beginningOfDocument)
+            } else if selectedTextRange != nil {
```

Comment:
Nit: Instead of checking if the value is not nil then proceed to optionally and explicitly unwrap the value below, you can set this to be `if let range = selectedTextRange` and use that below without the ? or !.